cli/dcos-spark/submit_builder.go

```diff
@@ -280,75 +292,87 @@ func parseApplicationFile(args *sparkArgs) error {
 	return nil
 }
 
-func cleanUpSubmitArgs(argsStr string, boolVals []*sparkVal) ([]string, []string) {
-
-	// collapse two or more spaces to one.
-	argsCompacted := collapseSpacesPattern.ReplaceAllString(argsStr, " ")
+// we use Kingpin to parse CLI commands and options
+// spark-submit by convention uses '--arg val' while kingpin only supports --arg=val
```

Comment:
Could we create a ticket to address this. Kingpin supports *both* `--arg val` and `--arg=val`.

Reply:
here you go: https://jira.mesosphere.com/browse/DCOS-41107

Reply:
Great! Thanks.